requirements.txt

```diff
@@ -9,6 +9,7 @@ tblib
 toolz >= 0.7.4
 tornado >= 5
 zict >= 0.1.3
+jupyter-server-proxy == 1.1.0
```

Comment:
The strict pinning makes me concerned here.  Can we do `>=`?

Also, does this bring in all of Jupyter?  If so, we might want to make this optional.

Reply:
To be more explicit, strict pinning is awesome when making applications but leads to unhappy users when making libraries.  When multiple libraries use strict pinning then it becomes very hard to construct a valid environment.

Reply:
I opted for `==` because I wasn't sure of the dev cycle and stability of jupyter-server-proxy and thus was being conservative (things work now) and we can afford to move slowly here.

It doesn't bring in all of Jupyter but it does bring in the notebook ~7MBs.  We could make this optional but it would me putting back some logic into worker-table.html or scheduler_html.py ?